sdk/opendp/whitenoise/synthesizers/pytorch/nn/patectgan.py

```diff
@@ -0,0 +1,378 @@
+import math
+import numpy as np
+import pandas as pd
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torch.utils.data import DataLoader, TensorDataset
+
+from ctgan import CTGANSynthesizer
+
+from .privacy_utils import weights_init, pate, moments_acc
+
+import numpy as np
+import torch
+from torch import optim
+from torch.nn import functional
+import torch.nn as nn
+import torch.utils.data
+from torch.nn import BatchNorm1d, Dropout, LeakyReLU, Linear, Module, ReLU, Sequential,Sigmoid
+from torch.nn import functional as F
+from opendp.whitenoise.synthesizers.base import SDGYMBaseSynthesizer
+
+import ctgan
+from ctgan.transformer import DataTransformer
+from ctgan.conditional import ConditionalGenerator 
+
+from ctgan.models import Generator
+
+from ctgan.sampler import Sampler 
+
+from ctgan import CTGANSynthesizer
+
+import torchdp
```

Comment:
also not used. 

Not sure why the CI is not running. If you can remove this import, run the tests locally and send me a screen shot I can verify the tests passed. 

I am looking into why the build is currently not firing

Reply:
![new_tests_patectgan](https://user-images.githubusercontent.com/60412438/92509385-2d180400-f1d8-11ea-920d-20b527d07699.PNG)

Reply:
just ran the 2 new tests that were added locally, should be good to merge